apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -318,9 +319,11 @@ export default function ListBoxInline({ app, fieldIdentifier, stateName = '$', o
           </Grid>
         </Grid>
       )}
-      <Grid item ref={searchContainerRef}>
-        <ListBoxSearch model={model} dense={dense} keyboard={keyboard} visible={searchVisible} />
-      </Grid>
+      {!disableSearch && (
+        <Grid item ref={searchContainerRef}>
```

Comment:
This part looks ok, but i think maybe this could case issues on line https://github.com/mhs30/nebula.js/blob/34115f7c02e6ccba0699a3a4a731ffc64e414980/apis/nucleus/src/components/listbox/ListBoxInline.jsx#L223
while trying to access of `searchContainer.current`. i think it should throw a "trying to access the value of undefined" error if we fall in that case

Reply:
not actually, that if case is above the usage of `searchContainer.current`, i would rather say one more check for the value either in if case like: 
```tsx
   if ( searchContainer?.current && ((search && focusSearch) || (search === 'toggle' && showSearch))) {
       // ...
    }
```
or inside of it:
```tsx
    if ((search && focusSearch) || (search === 'toggle' && showSearch)) {
      const input = searchContainer?.current.querySelector('input');
      input && input.focus();
    }
```

I would go with 2nd solution since it already checks for `input` value first (incase of `searchContainer` being undefined -> `input` becomes undefined) and then focuses on.

Reply:
You won't get there due to the dependency array of the useEffect

Reply:
✅